src/main/kotlin/com/julianasaran/blog/application/list/ListAuthorsQuery.kt

```diff
@@ -0,0 +1,5 @@
+package com.julianasaran.blog.application.list
+
+class ListAuthorsQuery(val name: String) {
+    data class Response(val name: String)
```

Comment:
Lists should always contain the id, this is the information needed to load the entire resource
```suggestion
    data class Response(val name: String)
```